packages/defichain-testing/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "private": false,
+  "name": "@defichain/testing",
+  "version": "0.0.0",
+  "description": "A collection of TypeScript + JavaScript tools and libraries for DeFi Blockchain developers to build decentralized finance on Bitcoin",
+  "keywords": [
+    "DeFiChain",
+    "DeFi",
+    "Blockchain",
+    "API",
+    "Bitcoin"
+  ],
+  "repository": "DeFiCh/jellyfish",
+  "bugs": "https://github.com/DeFiCh/jellyfish/issues",
+  "license": "MIT",
+  "contributors": [
+    {
+      "name": "DeFiChain Foundation",
+      "email": "engineering@defichain.com",
+      "url": "https://defichain.com/"
+    },
+    {
+      "name": "DeFi Blockchain Contributors"
+    },
+    {
+      "name": "DeFi Jellyfish Contributors"
+    }
+  ],
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc"
+  },
+  "dependencies": {
+    "@defichain/jellyfish-network": "0.0.0",
+    "@defichain/jellyfish-crypto": "0.0.0"
+  },
+  "devDependencies": {
+    "@defichain/testcontainers": "0.0.0"
+  }
```

Comment:
this should be peer dependencies, peer dependencies implies it will be provided by the package user. Since the container will always be passed from the consumer, it's a good idea to put as peer deps.